ansible/roles/test/tasks/iface_naming_mode/show_queue_counters.yml

```diff
@@ -0,0 +1,36 @@
+- block:
+    - name: show queue counters {{intf}}
+      shell: sudo  show queue counters {{intf}} | grep "UC\|MC"
+      register: queue_counter
+
+    - debug: var=queue_counter.stdout_lines
+
+    - name: Check the {{mode}}  interface name is present in output when mode is set to {{mode}}
+      assert:
+        that:
+        - queue_counter.stdout |  search("{{intf}}\s+[U|M]C{{item}}\s+\d+\s+\d+\s+\d+\s+\d+")
+      with_sequence: start=0 end=9
+
+    - name: show queue counters for all interfaces
+      shell: sudo  show queue counters | grep "UC\|MC"
+      register: queue_counter
+
+    - debug: var=queue_counter.stdout_lines
+
+    - name: Check default interface name is present in output when mode is  is set to default 
+      assert:
+        that:
+        - queue_counter.stdout |  search("{{item}}\s+[UC|MC\d]+\s+\d+\s+\d+\s+\d+\s+\d+") and '{{port_name_map[item]}}' not in queue_counter.stdout
+      with_items: default_interfaces 
+      when: mode=='default'
+
+    - name: Check alias interface name is present in output when mode is set to alias
+      assert:
+        that:
+        - queue_counter.stdout |  search("{{item}}\s+[UC|MC\d]+\s+\d+\s+\d+\s+\d+\s+\d+") and 'Ethernet' not in queue_counter.stdout
```

Comment:
Checking for the string "Ethernet" is not reliable enough, as some vendor aliases also contain the string "Ethernet". The proper solution is to retrieve the appropriate port name and alias from the device's config and check against them explicitly.

Reply:
modified as per comments